### ISSUE TITLE
Can I make a MAF file using only one sample?

### ISSUE THREAD

**Issue by omG0-hub** [PoisonAlien/maftools#925]
Hello, 

I wonder that I possible to make a MAF file only using one sample.
also, I am qurious that single maf file can be used the maftools for visualizing onco plot?
when I saw the the "https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/#protected-maf-file-structure" web site, only Somatic (Normal & Tumor) MAF file can use the drawing onco plot image., is it true..?

Thank you,

**Comment by PoisonAlien** [comment]
Hi,
Of course you can read a single sample maf and plot the data. But most functions and analysis won't be so meaningful.

**Comment by omG0-hub** [comment]
Thank you for your response,
it really helps me cause, I could not find related information.

I have one more question.
I tried to make an oncoplot by using a maf file.
How can I make a maf file that contains multi-sample information??
I referred linked site,
http://bioconductor.org/packages/devel/bioc/vignettes/maftools/inst/doc/oncoplots.html
I think I used a maf file with 193 sample information, so I was wondering
how to make a maf file with these 193 sample file information.

Can I simply combine the vcf of a single sample and then convert it to maf??
I'd like to create an oncoplot for the results of the somatic pair
analysis, but I'm wondering how to make a maf file with various numbers of
sample information.





2023년 4월 8일 (토) 오후 9:12, Anand Mayakonda ***@***.***>님이 작성:
